### PR TITLE
Create user routes page

### PR DESCRIPTION
**Pull request by AndriiChepeliuk**
=============== feat/dv-107/create_user_routes_page ===============

--- Added ---

- page to view user`s routes;
- function to get address of start point and end point;

### CODE REVIEW

File: src/constants/api/urls.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ export const OFFERS_URLS = {
 
 export const TRIPS_URL = {
     CREATE: TRIP_URL + "/create",
-    GET_ALL_ROUTES: TRIP_URL + "/routes"
+    GET_ALL_ROUTES: TRIP_URL + "/routes",
+    GET_BY_USER: TRIP_URL + "/my-routes"
```
Comment (Grasmus) [inline src/constants/api/urls.js]: It`s about routes, not about trips

File: src/components/myRoutes/index.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <Pagination
+                        onChange={onPaginationChange}
+                        total={routes.totalItems}
+                        showSizeChanger
+                        showTotal={(total) => `Total ${total} items`}
+                        pageSizeOptions={customPageSizeOptions}
+                        defaultPageSize={paginationDefaultFilter.DEFAULT_SMALL_PAGE_SIZE}
+                    />
+                </div>
+
```
Comment (Grasmus) [inline src/components/myRoutes/index.jsx]: Remove empty line

**Review by Grasmus — APPROVED** [review]

File: src/App.js [hunk 3]
```diff
@@ -135,6 +136,14 @@ export default function App() {
                     <RoutesPage />
                 </PrivateRoute>
 
+                <PrivateRoute
+                exact
+                path="/my-routes"
+                allowedRoles={[userRoles.USER]}
```
Comment (marynakernychna) [inline src/App.js]: Unaligned code

File: src/api/trips.js [hunk 4]
```diff
@@ -11,4 +11,10 @@ export default class tripsService {
             `?PageNumber=${paginationFilterModel.pageNumber}
              &PageSize=${paginationFilterModel.pageSize}`);
     }
+
+    static getAllRoutesByUser(paginationFilterModel){
+        return instance.get(TRIPS_URL.GET_ROUTES_BY_USER +
+            `?PageNumber=${paginationFilterModel.pageNumber}
+             &PageSize=${paginationFilterModel.pageSize}`)
```
Comment (marynakernychna) [inline src/api/trips.js]: Add " ; " at the end of the line

File: src/components/myRoutes/myRoute/index.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                        <Tooltip placement="top" title={allCities}>
+                            {"Through: " + throughCities}
+                        </Tooltip>
+                    </p>
+
+                    <div className="innerBody">
+                        <div className="rightSide">
+                            <p className="dataField">
+                                <AiOutlineCar size={DEFAULT_ICON_SIZE} />
+                                {props.data.car.model}
```
Comment (marynakernychna) [inline src/components/myRoutes/myRoute/index.jsx]: Give the user information about what you output, for one icon does not make it clear

File: src/components/myRoutes/myRoute/index.jsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+                            </p>
+
+                            <p className="dataField">
+                                <AiOutlineInfoCircle size={DEFAULT_ICON_SIZE} />
+                                {props.data.car.registrationNumber}
```
Comment (marynakernychna) [inline src/components/myRoutes/myRoute/index.jsx]: Give the user information about what you output, for one icon does not make it clear

File: src/components/myRoutes/myRoute/index.jsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+                            </p>
+
+                            <p className="dataField">
+                                <GiWeight size={DEFAULT_ICON_SIZE} />
+                                {props.data.loadCapacity + " kg"}
```
Comment (marynakernychna) [inline src/components/myRoutes/myRoute/index.jsx]: Give the user information about what you output, for one icon does not make it clear

File: src/components/myRoutes/myRoute/index.jsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+                                    <div className="date">
+                                        <AiOutlineArrowRight size={DEFAULT_ICON_SIZE} />
+                                        {moment(props.data.expirationDate).format('LLL')}
+                                    </div>
+                                </div>
+                            </p>
+
+                            <p className="dataField">
+                                <RiPinDistanceLine size={DEFAULT_ICON_SIZE} />
+                                {props.data.distance + " km"}
```
Comment (marynakernychna) [inline src/components/myRoutes/myRoute/index.jsx]: Give the user information about what you output, for one icon does not make it clear

File: src/services/trips.js [hunk 9]
```diff
@@ -31,6 +31,32 @@ export async function getAllRoutes(paginationFilterModel) {
...
+    }
+  )
+  .catch(() => {
+    errorMessage(
+      tripsMessages.LOAD_ROUTES_FAILED,
+      generalErrorMessages.SOMETHING_WENT_WRONG
+    );
+  });
+}
+
```
Comment (marynakernychna) [inline src/services/trips.js]: Unaligned code

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**